generic/tests/cfg/boot_vm_in_hugepage.cfg

```diff
@@ -9,16 +9,24 @@
     extra_params += " -mem-path /mnt/kvm_hugepage"
     pre_command_noncritical = yes
     pre_command = "echo 3 > /proc/sys/vm/drop_caches"
+    s390x:
+        kvm_module_parameters = 'hpage=1'
     variants:
+        - 1M:
```

Comment:
Is 1M huge page only supported on s390x ? If so, we need to set it "only s390x", thanks.